example_project/rgd_example/settings.py

```diff
@@ -46,6 +48,10 @@ def before_binding(configuration: ComposedConfiguration) -> None:
     # To use endpoints from external origin
     CORS_ORIGIN_ALLOW_ALL = True
 
+    CELERY_TASK_TIME_LIMIT = values.IntegerValue(
+        environ=True, default=timedelta(days=1).total_seconds()
+    )
```

Comment:
Actually, can you place this in `ResonantGeoDataBaseMixin` in `django-rgd/rgd/configuration/configuration.py`